packages/connect/src/core/index.ts

```diff
@@ -428,6 +428,8 @@ const inner = async (context: CoreContext, method: AbstractMethod<any>, device:
         sendCoreMessage(createPopupMessage(POPUP.CANCEL_POPUP_REQUEST));
     }
 
+    method.checkDeviceCapability();
```

Comment:
I am just thinking what is the line to place this check. There are other similar checks, for example this one https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/core/index.ts#L214
maybe @marekrjpolak has a stronger opinion?